tests/cnf/ran/internal/ranhelper/version.go

```diff
@@ -0,0 +1,211 @@
+package ranhelper
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/deployment"
+	"github.com/openshift-kni/eco-goinfra/pkg/olm"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/raninittools"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/ranparam"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// InitializeSpokeNames initializes the name of spoke 1 and, if present, spoke 2.
+func InitializeSpokeNames() error {
+	var err error
+
+	// Spoke 1 is required to be present.
+	ranparam.Spoke1Name, err = getClusterName(os.Getenv("KUBECONFIG"))
```

Comment:
Just as an overall comment, not necessarily something wrong with this functionality of this PR because I think this "works" fine but global variables make things near impossible to trace and to unit test if needed in the future.  These funcs that set `ranparam` values should be using parameters and return values.

It's probably fine for this PR since this is pretty much repo-wide at this point but I would love to see a better way of doing this in the future.

Reply:
I have to agree that the global mutable state is pretty non ideal. For this PR I'll go ahead and implement Yang's suggestion which will centralize the global state into the config. This should organize things a bit more if we ever do want to make an effort to move away from global variables everywhere